client/src/components/Header.jsx

```diff
@@ -9,6 +9,7 @@ import {
   Popover,
   MenuItem,
   Badge,
+  Link as MuiLink,
```

Comment:
It's a good idea to rename it so it doesn't conflict with the `react-router-dom` link, but are you using it?

Reply:
Shouldn't be there anymore, but I was renaming it because we were also importing `Link` from react-router.